tests/streamlit_ui_tests/add_column_tests.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { FrameLocator, Page, expect, test } from '@playwright/test';
+import { getMitoFrameWithTypeCSV, createNewColumn, clickButtonAndAwaitResponse } from './utils';
+
+
+test('Add new column to end using context menu', async ({ page }) => {
+    const mito = await getMitoFrameWithTypeCSV(page);
+    await createNewColumn(page, mito, 3, 'Column4');
```

Comment:
You're not actually checking the column exists after! There's utilities in the pivot PR that make this super easy.

Reply:
Applies to all of these tests